akka-operator/values.yaml

```diff
@@ -40,3 +40,7 @@ affinity: {}
 # - gcp
 provider:
   name: false
+
+gcp:
+  repository: gcr.io/cloud-marketplace/lightbend-public/akka-cloud-platform
+  deployer: false
```

Comment:
switch to distinguish between manual Helm-based installation and Marketplace automatic one.